### ISSUE TITLE
Removing the mmCif file parsing alert from stdout

### ISSUE THREAD

**Issue by chmnk** [schrodinger/pymol-open-source#331]
Hi all,

Do you think if it would be possible to hide the following output 
```C++
PRINTFB(G, FB_Executive, FB_Details)
      " ExecutiveLoad-Detail: Detected mmCIF\n" ENDFB(G);"
```
from the https://github.com/schrodinger/pymol-open-source/blob/c3e298356a13f93fce939f860800625272754c97/layer2/CifMoleculeReader.cpp#L1116?

I like using `pymol` as a simplistic `PDB/CIF` parser, but this output spoils the log a bit when working with thousands of files.

I am not sure how crucial is this print (and the small-mol-cif one below), and what is the best strategy of removing it (maybe changing the logging level to something else?), so no pull request from me so far.

**Comment by JarrettSJohnson** [comment]
Hi @chmnk , I'll address this in a couple of hours. This is already recently available in incentive if the `quiet` argument is supplied, but that didn't seem to make its way to open-source.

**Comment by JarrettSJohnson** [comment]
Resolved via 9c82294fab18d537b3b14c00767bdb7d126aea2c
